assets/source/catalog-sync/sections/SyncState.test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import SyncState from './SyncState';
+
+describe( 'SyncState component', () => {
+	test( 'should render header and footer correctly', () => {
+		const { getByText, getByRole } = render( <SyncState /> );
+
+		expect( getByRole( 'heading' ) ).toHaveTextContent( 'Overview' );
+
+		expect(
+			getByText( 'Set up and manage ads to increase your reach with' )
+		).toBeTruthy();
```

Comment:
💅 

Since you're testing in L22 already the text inside the link, I would remove the extra query here testing the rest of the text.

Reply:
You're right, fixed in b3e42d2, thanks!